samples/SkiaMultiplatformSample/build.gradle.kts

```diff
@@ -91,11 +92,15 @@ kotlin {
         binaries.executable()
     }
 
+    wasm {
+        browser()
+        binaries.executable()
+    }
+
     sourceSets {
         val commonMain by getting {
             dependencies {
                 implementation("org.jetbrains.skiko:skiko:$version")
-                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion")
```

Comment:
Is it a redundant dependency (not needed at all in this example)?

Reply:
Well, I do not see any coroutine code in this sample.